src/test/ui/argument-suggestions/basic.stderr

```diff
@@ -0,0 +1,62 @@
+error[E0059]: one or more arguments to this function are incorrect
+  --> $DIR/basic.rs:31:5
+   |
+LL |     invalid(1.0);
+   |     ^^^^^^^^---^
+   |             |
+   |             the type of this argument is incorrect
```

Comment:
Yea, I wasn't quite sure how to get human readable names for the error message, and when I tried looking into the existing code it became a very deep rabbit hole and I quickly got lost.  Any tips on methods I could reuse to avoid a ton of spurious work?